include/circt/Dialect/HW/HWVisitors.h

```diff
@@ -33,7 +33,9 @@ class TypeOpVisitor {
                        // Struct operations
                        StructCreateOp, StructExtractOp, StructInjectOp,
                        // Cast operation
-                       BitcastOp, ParamValueOp>([&](auto expr) -> ResultType {
+                       BitcastOp, ParamValueOp,
```

Comment:
Are these changes necessary for this PR? Seems like these are only necessary for ExportVerilog support.